### ISSUE TITLE
Question about the metrics of Table 4

### ISSUE THREAD

**Issue by Hugo-cell111** [Gavinwxy/DGCL#9]
Hi! Could you provide the code of the metrics(Silhouette, Calinski-Harbasz and Davies-Boulding) shown in Table 4 in your paper? Thanks a lot!

**Comment by gavinwxy** [comment]
Hi, thanks for your interest. The evaluation codes are unfortunately lost during code cleanning. They are implemented based on sklearn library, for example, the [silhouette score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.silhouette_score.html) and [Calinski and Harabasz score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.calinski_harabasz_score.html). Before evaluation, I collected pixel-wise features from all categories and balanced their quantity. All features are evaluated with their pseudo labels.
